operator/pkg/apis/operator/v1alpha1/type.go

```diff
@@ -449,18 +453,40 @@ type KarmadaSearch struct {
 	// CommonSettings holds common settings to karmada search.
 	CommonSettings `json:",inline"`
 
-	// ExtraArgs is an extra set of flags to pass to the karmada-descheduler component or override.
+	// ExtraArgs is an extra set of flags to pass to the karmada-search component or override.
 	// A key in this map is the flag name as it appears on the command line except without
 	// leading dash(es).
 	//
-	// Note: This is a temporary solution to allow for the configuration of the karmada-descheduler
+	// Note: This is a temporary solution to allow for the configuration of the karmada-search
 	// component. In the future, we will provide a more structured way to configure the component.
 	// Once that is done, this field will be discouraged to be used.
 	// Incorrect settings on this field maybe lead to the corresponding component in an unhealthy
 	// state. Before you do it, please confirm that you understand the risks of this configuration.
 	//
 	// For supported flags, please see
-	// https://github.com/karmada-io/karmada/blob/master/cmd/descheduler/app/options/options.go
+	// https://github.com/karmada-io/karmada/blob/master/cmd/search/app/options/options.go
```

Comment:
Just a question, Shall we link these options to https://karmada.io/docs/reference/components/karmada-metrics-adapter?

Just feel that the reference on the website is more clear.

Reply:
Indeed, I'm with you.